### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Ferizad**
- Implement the design only (JSX and CSS)✅
- Use functional components✅
- Create at least 1 child component✅
- Once the Calculator is ready, import it and use it in your top-level component: App.js✅
- Test it in the browser; it should render without problems.✅

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.App {
```
Comment (rustam-tolipov) [inline src/App.css:1]: - [ ] As the requirement given we need to delete all. I can see that this app css is the base styling that need to be removed. Kindly remove it :)

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (rustam-tolipov) [inline src/App.test.js:8]: [OPTIONAL] You are free to remove this as well :)

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
```
Comment (rustam-tolipov) [inline src/index.css:27]: - [ ] This one is also need to be remove. Please remove it as well :)

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <div className="btn orng">X</div>
+        <div className="btn">4</div>
+        <div className="btn">5</div>
+        <div className="btn">6</div>
+        <div className="btn orng">+</div>
+        <div className="btn">1</div>
+        <div className="btn">2</div>
+        <div className="btn">3</div>
+        <div className="btn orng">-</div>
+        <div className="btn">0</div>
```
Comment (rustam-tolipov) [inline src/components/Calculator.js]: - [ ] Your styling a bit different from the original to make it more matchable. Please, accept my suggestions :)

```suggestion
        <div className="btn zero">0</div>
```

File: src/components/Calculator.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  height: 100%;
+  width: 100%;
+  gap: 5px;
+}
+
+.buttons {
+  grid-column: 1/ -1;
+  text-align: center;
+  padding: 1rem;
+  font-size: 2rem;
```
Comment (rustam-tolipov) [inline src/components/Calculator.css:35]: Suggestion to make your styling better:

```suggestion
  font-size: 2rem;
  background: grey;
  border-radius: 8px;
```

File: src/components/Calculator.css [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.btn {
+  display: grid;
+  place-content: center;
+  background-color: rgb(180, 174, 174);
+  border-radius: 10%;
+  color: black;
+}
+
```
Comment (rustam-tolipov) [inline src/components/Calculator.css:47]: - [ ] The last suggestion :). DO NOT FORGET TO SHARE THE SCREENSHOT AFTER SUGGESTIONS!!!


```suggestion
.zero {
  grid-column: 3/ 1;
  border-radius: 8px;
}
```

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Ferizad,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Ferizad,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**